packages/chakra-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -56,11 +53,11 @@ export default function BaseInputTemplate<
       isReadOnly={readonly}
       isInvalid={rawErrors && rawErrors.length > 0}
     >
-      {displayLabel ? (
+      {displayLabel && !!label && (
```

Comment:
Since label can be an empty string, I use the `!!label` so that it renders `false` everywhere in the `chakra-ui` theme